mongodbatlas/resource_mongodbatlas_cluster_test.go

```diff
@@ -12,51 +12,226 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 	matlas "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
 	"github.com/mwielbut/pointy"
+	"github.com/spf13/cast"
 )
 
 func TestAccResourceMongoDBAtlasCluster_basicAWS(t *testing.T) {
 	var cluster matlas.Cluster
 
 	resourceName := "mongodbatlas_cluster.test"
 	projectID := os.Getenv("MONGODB_ATLAS_PROJECT_ID")
-	name := fmt.Sprintf("test-acc-%s", acctest.RandString(10))
+	tiers := []string{"M10", "M20", "M30", "M40"}
```

Comment:
For cost it might make sense to just do M10 and M30, those are the two levels where we have different capabilities.

Reply:
okay, thank you I will change the test case :)

Reply:
Sounds good!